src/bindings/jna/libelektra4j/src/main/java/org/libelektra/Key.java

```diff
@@ -489,10 +469,10 @@ public int copyMeta(final Key source, final String metaName) {
 	/**
 	 * Helper function to copy all meta information from a source key to this key
 	 *
-	 * @param source
-	 *            Key object that is used as source
-	 * @return 1 if meta was successfully copied, 0 if source doesn't contain any meta and nothing had to be done, -1 in case of an error or
-	 *         if the source parameter was null
+	 * @param source Key object that is used as source
+	 * @return 1 if meta was successfully copied, 0 if source doesn't contain any meta and nothing had to be done, -1
```

Comment:
should be @retval

Reply:
@*RetVal* is not a standard javadoc annotation (https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html). Do you still want it nonetheless?

Reply:
This is parsed by doxygen, so I do not see why javadoc is relevant here. Or do you want to build this with javadoc instead?